frontend/src/views/Layout/DashboardNavbar.vue

```diff
@@ -120,5 +70,10 @@ export default {
       this.$store.dispatch('logout')
     },
   },
+  beforeDestroy() {
+    if (this.$sidebar.showSidebar) {
+      this.$sidebar.showSidebar = false
+    }
```

Comment:
Isn't this solved by the two different layouts? Do we need those lines? What do they do?